File: FA-16185/SE6-PY/ejercicio1.py
num = int(input("Ingrese un numero:"))

while num <=0:
    num =int(input("Numero invalido.  Ingrese un numero valido: "))

i=1
print()
while i<=12:
    print(f"{num}x{i}={num*1}")
    i+=1

File: FA-16185/SE7-CS/EJERCICIO1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SE7_CS
{
    internal class EJERCICIO1
    {
        static void Main(string[] args)
        {
            int num,par=0,impar=0;
            do
            {

                Console.Write("Ingrese numero positivo (negativo para salir): ");
                num = int.Parse(Console.ReadLine());
                if (num != 0)
                {
                    if (num % 2 == 0) par++;
                    if (num % 2 !=0) impar++;
                }
                
            } while (num > 0);
            Console.WriteLine("\nCantidad de pares: "+par);
            Console.WriteLine("cantidad de impares: "+impar);
        }
    }
}
